tools/fuzzers/CMakeLists.txt

```diff
@@ -3,5 +3,17 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-add_subdirectory(libfuzzer)
+# Check LPM installation
+if (HERMES_ENABLE_LPMFUZZERS)
+    if (EXISTS ${PROJECT_SOURCE_DIR}/tools/fuzzers/utils/LPM)
+        message(STATUS "LPM already installed")
+        set(FUZZING_UTILS_DIR ${PROJECT_SOURCE_DIR}/tools/fuzzers/utils)
+    else()
+        message(FATAL_ERROR "LPM is not available, run the
+            script in tools/fuzzers/utils/prepare-env.sh to install it")
+    endif()
+endif()
```

Comment:
Can you instead use `find_package` from CMake to look for this? That's the best way to look for third-party dependencies. If not found you should just skip adding `fuzzer-json`